src/Test.ConcurrencyTests/rewrite.coyote.Windows.Release.json

```diff
@@ -0,0 +1,4 @@
+{
+  "AssembliesPath": "..\\bin\\Release\\netcoreapp3.1",
```

Comment:
>.. [](http://example.com/codeflow?start=21&length=2)

the release file has two dot while debug file has one

Reply:
@shaopeng-gh good catch. Fixed.